### DISCUSSION TITLE
CMS NodeJS

### DISCUSSION THREAD

**Discussion by renanlara** [frontendbr/forum#1161] [Discussão]
Galerinha,
Eu estava pensando aqui e cheguei até a achar algumas coisas bacanas. Mas, sempre procuro saber a opinião de vocês...

Alguém trabalha ou já trabalhou com algum tipo de CMS em Node? Se sim, quais as experiências?

Grato.

**Comment by kivervinicius** [comment]
Também é algo que sempre bato nessa tecla, acho algum que parece legal, desisto no meio do projeto e faço do zero com express e whatever. atualmente conheci o https://strapi.io mas não me satisfez também

**Comment by kavalcante** [comment]
Ah, tem o http://keystonejs.com

Porém, na minha opinião, CMS ainda é Wordpress. E se o projeto for tão complexo a ponto de precisar de algo mais completo, porém com velocidade ainda, eu utilizaria o Sails.js ou o Adonis.js.

**Comment by klarkc** [comment]
da uma olhada nesse site https://headlesscms.org/

parece ter alguns legais

**Comment by didiraja** [comment]
Sou meio amador em back e node, mas sempre me pergunto: a que se deve ainda nao termos uma ferramente decente e ampla de CMS no Node?

Pra mim é só o que falta pra nunca mais pensar em tocar em Apache/PHP. Wordpress é muito bom e cumpre seu papel, mas adoraria uma solução com Javascript.

**Comment by renanlara** [comment]
@didiraja, é o que eu realmente penso. Isso se não acontecer de fazerem com o wordpress.org o mesmo que fizeram com o .com, tudo em JS.

**Comment by lu-zen** [comment]
https://developer.wordpress.com/calypso/

**Comment by barraponto** [comment]
Já tem, se chama Ghost
https://github.com/TryGhost/Ghost https://ghost.org/
